src/nti/contentfragments/html.py

```diff
@@ -144,6 +146,12 @@ def __init__(self, *args, **kwargs):
             'style',
             'xml:lang'
         ])
+
+        allowed_attr_provider = component.queryUtility(IAllowedAttributeProvider)
+        if allowed_attr_provider:
+            additional_attrs_allowed = frozenset(allowed_attr_provider.allowed_attributes)
+            allowed_attributes = allowed_attributes | additional_attrs_allowed
```

Comment:
If the attributes from the utility are going to be additive, we should probably make that clear in the interface docs.